### ISSUE TITLE
ocean_volume vs. sea_water_volume

### ISSUE THREAD

**Issue by taylor13** [cf-convention/discuss#180]
What is the difference between "ocean_volume" and "sea_water_volume".  In the standard names listing, the note for sea_water_volume says "The quantity with standard name "sea_water_volume" is the total volume of liquid seawater in the global oceans, including enclosed seas."
There is no information provided regarding "sea_water_volume".  Should these two names be declared aliases for the same thing?

**Comment by JonathanGregory** [comment]
I agree that `òcean_volume` should be an alias of `sea_water_volume` and therefore `ocean_volume_fraction` should be renamed (made an alias) of `sea_water_volume_fraction`. I think the latter also makes sense because sea water is the name of the medium from which the ocean is composed.
